pages/services/storage/volume-plugins/index.md

```diff
@@ -8,9 +8,9 @@ excerpt:
 
 # Volume Plugins
 
-The Container Storage Interface (CSI) is an open specification developed by various container orchestrators through interaction with the open source community and storage vendors. Different CSI plugins expose CSI-compliant APIs on top of existing storage solutions such as those provided by storage vendors (e.g., Amazon EBS) as well as technologies like LVM2. 
+The Container Storage Interface (CSI) is an open specification developed by various container orchestrators through interaction with the open source community and storage vendors. Different CSI plugins expose CSI-compliant APIs on top of existing storage solutions such as those provided by storage vendors (e.g., Amazon EBS) and technologies like LVM2. 
 
-In order to provide a smooth configuration experience with DC/OS every CSI plugin you want to use requires a small bit of plugin-specific config code to be loaded by the DSS. This code, combined with the CSI plugin itself, is called a Volume Plugin. In future we hope to bundle a generic one with the DSS so any CSI plugin can be used.
+In order, to provide a smooth configuration experience with DC/OS every CSI plugin you want to use, requires a small bit of plugin-specific config code to be loaded by the DSS. This code combined with the CSI plugin itself, is called a Volume Plugin. In future, we hope to bundle a generic one with the DSS so any CSI plugin can be used.
```

Comment:
1. no comma after "In order" please
2. s/In future,/In the future/ (add "the" and remove the comma)